mobile/src/main/java/ch/epfl/sdp/mobile/state/StatefulVisitedProfileScreen.kt

```diff
@@ -82,27 +82,29 @@ fun StatefulVisitedProfileScreen(
     uid: String,
     onMatchClick: (ChessMatchAdapter) -> Unit,
     onPuzzleClick: (PuzzleInfoAdapter) -> Unit,
+    onBack: () -> Unit,
```

Comment:
Missing documentation.

Reply:
Updated doc in [02a8f6a](https://github.com/epfl-SDP/android/pull/303/commits/02a8f6adccec26f453c8dbc6542da6ab4097754b)